stories/grid/padding.js

```diff
@@ -0,0 +1,62 @@
+// @flow
```

Comment:
This is an example to showcase the new padding behavior, looks like this:

![image](https://user-images.githubusercontent.com/3877773/27395445-afce660e-5664-11e7-9376-7152deafb2ae.png)